testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -612,6 +612,33 @@ public void testMultiDelete(HiveOrPresto inserter, HiveOrPresto deleter)
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
+    public void testMultiInsertWithDelete()
```

Comment:
Maybe add a javadoc sth like

```
Test reading from a table after Hive multi-insert. Multi-insert involves non-zero statement ID, encoded
within ORC-level bucketId field, while originalTransactionId and rowId can be the same for two different rows.

Reply:
Added in method body. And restructured the comment we already had a bit.